### PR TITLE
Adjust inputs for new TMA behavior

### LINKED ISSUE #242: ASR Assessments rely on sketchy tensor mechanics action behavior
## Reason
The ASR assessment cases rely on the TMA to create block unrestricted variables despite being block restricted itself. I.want to change that behavior and need to fix the inputs here.

## Design
Add variables explicitly.

## Impact
Saner inputs,

### PR DESCRIPTION
**Pull request by dschwen**
Closes #242

### CODE REVIEW

File: assessment/asr_validation/wald2017b/analysis/A1-biaxial.i [hunk 1]
```diff
@@ -15,432 +15,437 @@
...
-  [../]
+  []
 []
 
 [Modules/TensorMechanics/Master]
-  [./concrete]
+  [concrete]
     block = 1
     strain = FINITE
-    add_variables = true
```
Comment (bwspenc) [inline assessment/asr_validation/wald2017b/analysis/A1-biaxial.i:144]: So are we getting rid of `add_variables`?

Comment (dschwen) [reply assessment/asr_validation/wald2017b/analysis/A1-biaxial.i:144]: No. Just in inputs that use the line element action, which does not properly coordinate with the TMA. Going forward we might want to improve that action.

Comment (bwspenc) [reply assessment/asr_validation/wald2017b/analysis/A1-biaxial.i:144]: Oh, I see...

**Review by bwspenc — APPROVED** [review]

File: assessment/asr_validation/wald2017b/analysis/A1-biaxial.i (hunk 1 above)
Comment (bwspenc) [reply assessment/asr_validation/wald2017b/analysis/A1-biaxial.i:144]: So we have to remove it from both the TM master and the line element action if we're using them together.

Comment (dschwen) [reply assessment/asr_validation/wald2017b/analysis/A1-biaxial.i:144]: yeah

**Pull request merged**